app/assets/javascripts/components/suspected_plagiarism/suspected_plagiarism_handler.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import createReactClass from 'create-react-class';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import {
+  fetchSuspectedCoursePlagiarism,
+  sortSuspectedPlagiarism
+} from '../../actions/suspected_plagiarism_actions';
+import SuspectedPlagiarismList from './suspected_plagiarism_list';
+
+  const PossiblePlagiarismHandler = createReactClass({
```

Comment:
I had totally forgotten that we already had a view for showing recent possible plagiarism across the whole system. This could probably be done by re-using the existing component (which shows up at `/recent-activity/plagiarism`), although making a new component as you've done is also fine.

Reply:
I tried to use that component but it used a drawer in the list to show / hide all courses the revision belongs to which wouldn't make much sense in this case. The alternative would be refactor the code to only render the drawer if a flag is passed to it, (or maybe it does make sense to show the drawer). I don't mind going this route if that's a better fit.